test/pipelines/osm/test_osm_mapper.py

```diff
@@ -0,0 +1,232 @@
+                "internet_access": "wlan",
+                "internet_access:fee": "no",
+                "name": "Hotel Graf Eberhard",
+                "operator": "Hotel Graf Eberhard GmbH + Co. KG",
+                "stars": "4",
+                "tourism": "hotel",
+                "wheelchair": "yes"
+            },
+            "type": "node"
+        }
+
+        address: Address = map_address_osm(json_data, 1)
+        print(address)
+
+        self.assertEqual(1, address.station_id)
+        self.assertEqual('Bei den Thermen 2', address.street)
+        self.assertEqual('Bad Urach', address.town)
+        self.assertEqual(None, address.district_old)
+        self.assertEqual(None, address.district)
+        self.assertEqual(None, address.state)
+        self.assertEqual('DE', address.country)
+        self.assertEqual(None, address.gmaps_latitude)
+        self.assertEqual(None, address.gmaps_longitude)
+
+    def test_charging_mapping__plain_01(self):
+        json_data = {
+            "id": 2420152562,
+            "lat": 48.9277666,
+            "lon": 8.4709132,
+            "tags": {
+                "amenity": "charging_station",
+                "bicycle": "yes",
+                "capacity": "4",
+                "fee": "no",
+                "socket:schuko": "4",
+                "socket:type2:combo:output": "2.3 kW"
+            },
+            "type": "node"
+        }
+
+        charging: Charging = map_charging_osm(json_data, 1)
+        print(charging)
+
+        self.assertEqual(1, charging.station_id)
+        self.assertEqual(4, charging.capacity)
+        self.assertEqual([2.3], charging.kw_list)
+        self.assertEqual(2.3, charging.total_kw)
+        self.assertEqual(2.3, charging.max_kw)
+        self.assertEqual([], charging.ampere_list)
+        self.assertEqual([], charging.volt_list)
+        self.assertEqual(['AC Schuko'], charging.socket_type_list)
+        self.assertEqual(False, charging.dc_support)
+
+    def test_charging_mapping__amperage_01(self):
+        json_data = {
+            "tags": {
+                "amperage": "1.2",
+            },
+        }
+        charging: Charging = map_charging_osm(json_data, 1)
+        self.assertEqual([1.2], charging.ampere_list)
+
+    def test_charging_mapping__amperage_02(self):
+        json_data = {
+            "tags": {
+                "amperage": "22.6 kVA",
+            },
+        }
+        charging: Charging = map_charging_osm(json_data, 1)
+        self.assertEqual([22.6], charging.ampere_list)
+
+    def test_charging_mapping__amperage_03(self):
+        json_data = {
+            "tags": {
+                "amperage": "125 A - DC",
+            },
+        }
+        charging: Charging = map_charging_osm(json_data, 1)
+        self.assertEqual([125.0], charging.ampere_list)
+
+    def test_charging_mapping__amperage_04(self):
+        json_data = {
+            "tags": {
+`                "amperage": "16;32",
```

Comment:
The backticks in line 110 seem to be a typo.

Reply:
Fixed.